src/Core/Entity/Image.php

```diff
@@ -183,6 +202,7 @@ protected function saveToTemporaryFile()
         }
         $this->temporaryFile = TMP_DIR . uniqid("", true);
         file_put_contents($this->temporaryFile, $content);
+        $this->sourceMimeType = finfo_file(finfo_open(FILEINFO_MIME_TYPE), $this->temporaryFile);
```

Comment:
We don't know the bytedepth or compression mode with this. It only works for getting mime type. 
I guess it's fine for now.